.github/workflows/performance.yml

```diff
@@ -25,12 +25,16 @@ jobs:
     # Don't run on private repo unless it is a PR.
     if: (github.repository == 'Firebase/firebase-ios-sdk' && github.event_name == 'schedule') || github.event_name == 'pull_request'
     runs-on: macos-latest
+    strategy:
+      matrix:
+        target: [iOS, tvOS]
+        test: [unit, proddev]
```

Comment:
What is the difference between unit and proddev?

Reply:
See the corresponding changes in scripts/build.sh